cli/package.json

```diff
@@ -34,5 +40,25 @@
 		"node-fetch": "^2.6.7",
 		"pem-file": "^1.0.1",
 		"prompts": "^2.4.2"
+	},
+	"devDependencies": {
+		"@chialab/esbuild-plugin-meta-url": "^0.17.2",
+		"esbuild": "^0.17.5",
+		"pkg": "^5.8.0",
+		"shx": "^0.3.4"
+	},
+	"files": [
+		"dist/**/*"
+	],
+	"pkg": {
+		"compress": "GZip",
+		"assets": [
+			"dist/package.json"
+		],
+		"targets": [
+			"node18-linux",
+			"node18-macos",
+			"node18-win"
```

Comment:
Can we target node14? I think I saw someone using mops on node 14

Reply:
This setting specifies the version of Node which is bundled with the standalone binary (so this will work for any Linux/Mac/Windows platform regardless of Node version). 

Changing `"module"` to `"commonjs"` in the `package.json` file will make it possible to use MOPS on Node 14 (since ES modules are only supported in Node 16+).